daemon/config/config.go

```diff
@@ -160,7 +160,9 @@ type CommonConfig struct {
 	ExecRoot              string                    `json:"exec-root,omitempty"`
 	SocketGroup           string                    `json:"group,omitempty"`
 	CorsHeaders           string                    `json:"api-cors-header,omitempty"`
-	ProxyConfig
+
+	// ProxyConfig holds the proxy-configuration for the daemon.
+	ProxyConfig `json:"proxy-config"`
```

Comment:
FWIW, I was considering to have an `omitempty`, but that required the struct to be a pointer, and all the complexity involved with that, so I decided not to go down that road